src/sentry/static/sentry/app/views/settings/projectDebugFiles/index.tsx

```diff
@@ -131,37 +172,84 @@ class ProjectDebugSymbols extends AsyncView<Props, State> {
           </React.Fragment>
         )}
 
-        <TextBlock>
-          {t('This list contains all uploaded debug information files:')}
-        </TextBlock>
+        <Wrapper>
+          <TextBlock noMargin>{t('Uploaded debug information files')}:</TextBlock>
+
+          <Filters>
+            <Label>
+              <Checkbox
+                checked={showDetails}
+                onChange={e => {
+                  this.setState({showDetails: (e.target as HTMLInputElement).checked});
+                }}
+              />
+              {t('details')}
```

Comment:
It isn't clear on what will happen when this is clicked. Should this be 'show details'?

Reply:
updated 👍